src/common/dependencies.ts

```diff
@@ -21,6 +21,16 @@ export interface Dependency {
     description?: string;
 }
 
+const getOnnxRuntime = (canCuda: boolean) => {
```

Comment:
Could we change this function to return a full PyPiPackage instead? Since onnxruntime-silicon doesn't seem to be official, I worry that the version of it and the official package diverge eventually.

Basically, I think the function should be like this:
```
if m1: return { package: 'onnxruntime-silicon', ... }
return { ...previousPackageCode }
```

This clearly shows the difference between the two.

Also, onnxruntime-silicon is 6MB :)